components/LoginModal.vue

```diff
@@ -61,7 +56,7 @@
             data-testid="register-now-button"
             @click="setIsLoginValue(false)"
           >
-            Register now
+            Register Now
```

Comment:
use i18n lang helper instead of hardcoding the text

Reply:
Resolved